README.md

```diff
@@ -145,6 +147,24 @@ file_revisions = crowdin.list_file_revisions(your_file_id, { limit: 10 }, your_p
 # Note: more examples you can find in spec folder
 ```
 
+### Fetch All
+
+You also can fetch all records from paginatable methods using `fetch_all` method (where we have limit and offset in arguments).
+
+```ruby
+# FetchAll options:
+# * limit, Integer, default: 500 | How many records need to load per one request
+# * offset, Integer, default: 0
+# * request_delay, Integer (seconds), default: 0 | Delay between requests
+
+# Example and use cases:
+@crowdin.fetch_all(:list_projects)
```

Comment:
This interface works, but I have imagined it a little bit differently. I understand there can be different ways of solving the same task but I just want to share how I imagined it, just in case it helps in any way.

So I thought of this more like of how ActiveRecord works in Rails.

So for example we call `@crowdin.list_project` and it does not immediately return the response. Instead it returns a special object/enumerator. This object provides a certain interface that allows to get different subsets, e.g.:

```ruby
# Get just the first one:
@crowdin.list_projects.first

# Get all:
@crowdin.list_projects.all

# Limit and chain with offset:
@crowdin.list_projects.limit(42).offset(5)
```

ActiveRecord models actually make DB calls only when the enumeration started, so when actually `each` or some other method with the block is called, or `to_a` -- to explicitly request conversion of it to an Array. 

So we could chain methods this way and then only call the API once in the end in the lazy manner when we need the data.

I like this idea because it's very familiar to many Ruby developers and thus it provides the API in the most "native" way.

But as I said, it's just a certain way how I like it, it doesn't mean other approaches are wrong. In the end what's important is that people just need to get their job done.

Reply:
@cmrd-senya thanks, your solution looks good

Reply:
IMO, It's the more obvious way and will allow avoid misunderstanding and passing wrong methods (not list) to the `crowdin.fetch_all` method at least

Reply:
I need time to think about that implementation
It's force us to release Major version, because we thinking about changing main response
But I too like this way